testplan/report/testing/schemas.py

```diff
@@ -297,14 +288,12 @@ class Meta:
 
     @post_load
     def make_test_report(self, data, **kwargs):
-        status_override = data.pop("status_override", None)
```

Comment:
why are we removing all these pop lines?

Reply:
I guess that it is originally needed, but as code evolves (the constructor of those Report classes changes), the code of deserialization does not follow. So I refine the code and remove the redundant.